terragrunt/aws/cloudfront/certificates.tf

```diff
@@ -34,6 +34,24 @@ resource "aws_route53_record" "simplify_privacy_statement_dns_validation" {
   ttl = 60
 }
 
+
+resource "aws_acm_certificate" "simplify_privacy_statement_fr_certificate" {
```

Comment:
Can you also update `aws_route53_record.simplify_privacy_statement_fr_dns_validation` resource to reference this new cert.